charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -498,6 +498,30 @@ spec:
             - name: cni-log-dir
               mountPath: /var/log/calico/cni
               readOnly: true
+        {{- if and .Values.config.birdExporter.enabled (not .Values.config.nonPrivileged) }}
```

Comment:
I guess we also need to check for `.Values.config.monitoring.enabled` as the port is defined there similarly to the check in `[bird-monitoring-service.yaml](https://github.com/gardener/gardener-extension-networking-calico/pull/687/files#diff-e235f325163ebd134ab429f6eef255a49b4e09268aca62f49efe15499722bfdf)`.